Makefile

```diff
@@ -58,20 +58,25 @@ bin/aws-fsx-csi-driver: | bin
 .PHONY: all
 all: all-image-docker
 
+# Builds all images and pushes them
```

Comment:
Nit: it builds the images *and manifest*. Could you change this to `Builds all images, and the manifest and push them to the public ECR`

Reply:
I think it's fine as is, image manifest seems to be implicit with images